scripts/restoreconfigdb.sh

```diff
@@ -2,6 +2,7 @@
 set -x
 
 S3_PATH=$1
+PGPW=`kubectl get secret kfuse-pg-credentials -o jsonpath="{.data.postgresql-password}" | base64 --decode ; echo`
```

Comment:
log an error and exit if PGPW is empty - can happen if the secret does not exist.